### ISSUE TITLE
Add Center for Scripture Engagement logo to site.

### ISSUE THREAD

**Issue by nurkkala** [quantum-bits/galilee-webapp#34]

**Comment by knkiers** [comment]
I don't think we currently have a version of the graphic that would match the colour scheme of the web site.  Maybe we need one that would be white in the foreground with a transparent background, or white in the foreground with something that matches the colour scheme for the background.

**Comment by knkiers** [comment]
Should this be moved to the beta release maybe?
